src/gui/entry/EntryModel.cpp

```diff
@@ -295,6 +297,18 @@ QVariant EntryModel::data(const QModelIndex& index, int role) const
     return QVariant();
 }
 
+QColor EntryModel::getColorFromString(QString colorStr)
```

Comment:
`QColor` can be directly constructed from string, see [this](https://doc.qt.io/qt-5/qcolor.html#QColor-5) and [this](https://doc.qt.io/qt-5/qcolor.html#setNamedColor).

Reply:
nice find! I looked at the `QColor` doc but thought the only string conversion function was from RGBA, and the string value stored in the database file is RGB.